modules/ROOT/pages/configuring-user-analytics.adoc

```diff
@@ -0,0 +1,68 @@
+= Configuring User Analytics 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use Google Analytics to gain valuable insights into how consumers are using your APIs and portal, identify areas for improvement, make data-driven decisions to optimize the user experience, and drive business success.
+
+The following data is tracked:
+
+API usage metrics: 
+
+* API instance access requests
+* API specification downloads
+* Most viewed API
+* Total number of calls made to an AP
+* Total number of APIs published in the portal 
```

Comment:
We don't have this metric, we should remove this line

Reply:
removed